src/runners/handlers/slack.py

```diff
@@ -163,10 +178,9 @@ def handle(
         log.debug(f'Slack response', response)
 
         if response['ok'] is False:
-            log.error(f"Slack handler error", response['error'])
-            return None
+            raise RuntimeError(f"Slack handler error {response['error']}")
 
         if 'message' in response:
             del response['message']
 
-    return response
+    return response
```

Comment:
don't strip trailing newlines

Reply:
Sorry, didn't get this.

Reply:
there is a newline at the end of the file that you took out in this PR -- please put it back